cmd/kk/pkg/bootstrap/os/tasks.go

```diff
@@ -172,6 +172,15 @@ var (
 		"ipvsadm -C",
 		"ip link del kube-ipvs0",
 		"ip link del nodelocaldns",
+		"ip link del cni0",
+		"ip link del flannel.1",
+		"ip link del flannel-v6.1",
+		"ip link del flannel-wg",
+		"ip link del flannel-wg-v6",
+		"ip link del cilium_host",
+		"ip link del cilium_vxlan",
+		"ip -br link show | grep 'cali[a-f0-9]*' | awk -F '@' '{print $1}' | xargs -r -t -n 1 ip link del",
+		"ip netns show 2>/dev/null | grep cni- | xargs -r -t -n 1 ip netns del",
```

Comment:
Can you paste more information about these 2 commands? Like some documents or screenshots.

Reply:
Add some docs about k8s network configuration and description for `delete-cluster` and `delete-node` command. Some explanations may not be accurate, please feel free to revise them.

Reply:
Thanks!